examples/natmod/stm32wb55_transparent_vcp/Makefile

```diff
@@ -0,0 +1,46 @@
+# This module is intended for use on stm32wb55 microcontrollers
+# and shows how a native module can be used to extend hardware
+# functionality.
+
+# Note: `pip install pyelftools` is required to build native modules.
+
+# Location of top-level MicroPython directory
+MPY_DIR = ../../..
+
+# Name of module
+MOD = transparent_vcp
+
+# Source files (.c or .py)
+SRC = \
+ _stm32wb55_transparent.c \
```

Comment:
remove the leading underscore from the name?

Reply:
Thought it might be useful to differentiate that this is intended to be "internal" as opposed to the `.py` file being the public interface, like the C accelerator modules on cpython. The implemented / exposed C module no longer matches the filename anyway so yeah the underscore isn't really useful here.